schema.sql

```diff
@@ -5,5 +5,6 @@ CREATE TABLE animals (
   date_of_birth DATE, 
   escape_attempts INT, 
   neutered BOOLEAN, 
-  weight_kg DECIMAL
+  weight_kg DECIMAL,
+  species CHAR(40)
```

Comment:
👍 Great progress on your project! However, there's an important adjustment needed to align with the project requirements:

- [ ] You should add the `species` column to the `animals` table by altering it, rather than adding it in the initial table creation. As mentioned earlier, move the alter statement to the `schema.sql` file. Additionally, it's advisable to remove line number 9 from the `schema.sql` file because having the `species` column in the initial creation and then altering it in the same file can lead to errors. The recommended approach is to delete the `species` column from the initial creation and add it using an `alter` statement as per the project requirements.

By making this adjustment, you'll ensure that your project structure aligns perfectly with the specified requirements. Keep up the good work! 👍📋🛠️